docs(frontend): massive modifications in the documentation [part 1] (#944)
zama-ai/concrete: docs/get-started/terminology.md

* **Circuit:** The result of compilation. A circuit includes both client and server components. It has methods for various operations, such as printing and evaluation.

> * **Table lookup (TLU):** Table loopup (TLU) stands for instructions of the form `y = T[i]`. In practice, in FHE, they are done with programmable bootstrapping, which is the equivalent operation on encrypted values.

Reviewer: ```suggestion
* **Table Lookup (TLU):** TLU stands for instructions in the form of `y = T[i]`. In the practice of FHE, this operation is performed with Programmable Bootstrapping, which is the equivalent operation on encrypted values.
```

Author: "In the practice of FHE"? it looks not to be what I meant and it looks strange

Author: @yuxizama tell me if we can change this, please

Reviewer: Do you mean "In Concrete, this operation is performed with Programmable Bootstrapping" ?
I just find "**In** practice, **in** FHE..." is a little bit repetitive

Author: Yes I agree it's repetitive but "In the practice of FHE" looks to have a different meaning to me. Let me propose something

Author: "* **Table Lookup (TLU):** TLU stands for instructions in the form of `y = T[i]`. In FHE, this operation is performed with Programmable Bootstrapping, which is the equivalent operation on encrypted values."

Good for you @yuxizama ?

Reviewer: Yes perfect, thank you!

> * **Programmable bootstrapping (PBS):** Programmable bootstrapping (PBS) are the equivalent of table lookup `y = T[i]` on encrypted values, i.e., where input `i` and output `y` are encrypted (but not the table `T`).

Reviewer: ```suggestion
* **Programmable Bootstrapping (PBS):** PBS is equivalent to looking up the table `y = T[i]` on encrypted values, which means that the inputs `i` and the outputs `y` are encrypted, but the table `T` is not encrypted.
```

Author: "looking up the table" looks strange to me to. "PBS is equivalent to table lookup" would be ok for me

Author: @yuxizama tell me if we can change this, please

Author: done
